ECE: Empty state and optional postal code support for addresses in the Gulf countries (#4797)
woocommerce/woocommerce-gateway-stripe: changelog.txt

* Fix - Ensure Amazon Pay, Apple Pay, and Google Pay display settings are managed correctly
* Dev - Add logging with DNS resolution diagnostics for URL validation issues when calling Stripe API
* Fix - Allow payment methods to be disabled when they are not available

> * Fix - Add express checkout support for Gulf countries (UAE, Bahrain, Kuwait, Oman, Qatar) with optional state and postal code

Reviewer: Nit: I think it would be clearer if we rephrase this to focus on the address aspects, as express checkout was technically supported before, so we are not "adding" support.
```suggestion
* Fix - Ensure state and postal code are optional in express checkout for Gulf countries (UAE, Bahrain, Kuwait, Oman, Qatar)
```
